scalyr_agent/config_main.py

```diff
@@ -1431,6 +1483,10 @@ def create_custom_dockerfile(
             # do find a need to take some action.
             sys.exit(finish_upgrade_tarball_install(paths[0], paths[1]))
 
+    if options.set_python is not None:
+        set_python_version(options.set_python)
+        print("Agent switched to {0}.".format(options.set_python))
```

Comment:
After this line (if the switch succeeded) we should probably exit, aka ``sys.exit(0)``, right?

Reply:
Yes, I think you right.